### DISCUSSION TITLE
I want to output data on the temporal changes in the position of a particle at a certain point.

### DISCUSSION THREAD

**Discussion by duanlinlim** [InteractiveComputerGraphics/SPlisHSPlasH#342] [General]
I want to output data on the temporal and positional changes of a particle at a certain point. How should I proceed. It seems that only PartioViewer can be used to view one by one.

**Comment by janbender** [comment]
You can also export the data and explore it in ParaView.

**Comment by duanlinlim** [comment]
Thank you for your response
Can you output a file that stores data and view it using Paraview? The specific values cannot be seen

**Reply by janbender** [reply]
Yes, just use the VTK exporter and define the particle attributes that you want to see.

**Comment by duanlinlim** [comment]
How to operate if you want to measure the water level change at position (2,0,0) in the Dam break experiment

**Reply by janbender** [reply]
The easiest way is to use a Python script that computes the value for you.
